docs/content/tutorial/default.yml

```diff
@@ -207,7 +207,7 @@ body:
         "name": "Nico Williams"
       }
       {
-        "message": "Fix iterration problem for non decimal string\n\nWhen the string transformation to number failed, all following\ntransformation failed too.\n\nThis happend because status in decNumberFromString function is\nupdated just in error case. Reusing the DEC_CONTEXT that failed\nbefore results into error even if the string is valid number.",
+        "message": "Fix iteration problem for non decimal string\n\nWhen the string transformation to number failed, all following\ntransformation failed too.\n\nThis happened because status in decNumberFromString function is\nupdated just in error case. Reusing the DEC_CONTEXT that failed\nbefore results into error even if the string is valid number.",
```

Comment:
Sigh, this probably isn't appropriate because the typo is burned into the repository, I'll drop it.